docs/contributor/concepts/error-output-format.md

```diff
@@ -0,0 +1,42 @@
+# Output Error Format
```

Comment:
ADR (architecture decision record) or ADL (architecture decision log) is well known name. Maybe we could use this name for directory `concepts`?